src/Plaid/PlaidClient.cs

```diff
@@ -165,6 +167,37 @@ public sealed partial class PlaidClient
 			}
 		}
 
+		public readonly async Task<FileResponse> ParseFileResponseAsync()
+		{
+			// NOTE: We do not dispose the HttpResponseMessage here. We pass this duty onto the FileResponse
+			var response = await Message.ConfigureAwait(false);
+			Logger.LogInformation("Completed file request. Url: {Url}, Status Code: {StatusCode}.", Url, response.StatusCode);
+
+			var status = response.StatusCode;
+			if (!response.IsSuccessStatusCode)
+			{
+				var json = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+				var error = ParseError((int)response.StatusCode, json);
+				var badresult = new FileResponse(status, error);
+				response.Dispose();
+
+				return badresult;
```

Comment:
Simplify code: 

```suggestion
				response.Dispose();

				return new FileResponse(status, error);
```

Reply:
Also, (question:) should there be a `ShowRawJson` check here to determine whether to include the raw json on the error?

Reply:
Aaah, yeah I guess so. I missed this in the latest updates.

Reply:
Also yeah, I'll update the simplification. I was going to debate about it, but I realized it's good the way you have it.